components/Footer.js

```diff
@@ -56,322 +56,383 @@ const Links = styled.div`
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Kashish_Garg.png',
+      name: 'Kashish Garg',
+      emoji: '🐒',
+      color: '#01DACC',
+      title: 'HackCamp Logistics Coordinator',
+      social: "https://www.linkedin.com/in/kashishgarg1/"
+    },
+    {
+      img: '/assets/profiles/Kevin_Gu.png',
+      name: 'Kevin Gu',
+      emoji: '🍏',
+      color: '#13B1C9',
+      title: 'Sponsorship Coordinator',
+      social: "https://www.linkedin.com/in/kevin-gu-/"
+    },
+    {
+      img: '/assets/profiles/Kitty_Liu.png',
+      name: 'Kitty Liu',
+      emoji: '🐈',
+      color: '#FFE633',
+      title: 'Hackcamp Logistics Coordinator',
+      social: "https://www.linkedin.com/in/kittyliu3"
+    },
+    {
+      img: '/assets/profiles/Lincoln_Lee.png',
+      name: 'Lincoln Lee',
+      emoji: '🦝',
+      color: '#2E2E54',
+      title: 'Development Coordinator',
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Linda_Ma.png',
+      name: 'Linda Ma',
+      emoji: '💫',
+      color: '#dcd0ff',
+      title: 'Engagement Director',
+      social: "https://www.linkedin.com/in/linda--ma"
+    },
+    {
+      img: '/assets/profiles/Lucas_Gingera.png',
+      name: 'Lucas Gingera',
+      emoji: '😶',
+      color: '#01DACC',
+      title: '1st Year Rep',
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Martin_Cai.png',
+      name: 'Martin Cai',
+      emoji: '🍵',
+      color: '#FFF8DC',
+      title: 'Co-President',
+      social: "https://www.martincai.xyz"
+    },
+    {
+      img: '/assets/profiles/Maureen_Luo.png',
+      name: 'Maureen Luo',
+      emoji: '🫶🏻',
+      color: '#c4d6bf',
+      title: 'Design Director',
+      social: "https://www.maureenluo.com/"
+    },
+    {
+      img: '/assets/profiles/Melvin_Teo.png',
+      name: 'Melvin Teo',
+      emoji: '💡',
+      color: '#003366',
+      title: 'Development Director',
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Michelle_Fung.png',
+      name: 'Michelle Fung',
+      emoji: '🤩',
+      color: '#D9E7E2',
+      title: 'Sponsorship Coordinator',
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Michelle_Kim.png',
+      name: 'Michelle Kim',
+      emoji: '🧸',
+      color: '#0DEFE1',
+      title: 'Project Manager',
+      social: "https://www.linkedin.com/in/michelle-yeojin-kim/"
+    },
+    {
+      img: '/assets/profiles/Michelle_Wang.png',
+      name: 'Michelle Wang',
+      emoji: '❄️',
+      color: '#9CDDFB',
+      title: 'Sponsorship Coordinator',
+      social: "https://www.linkedin.com/in/michelle-wang-mw/"
+    },
+    {
+      img: '/assets/profiles/Nicholas_Wong.png',
+      name: 'Nicholas Wong',
+      emoji: '😎',
+      color: '#01DACC',
+      title: 'HackCamp Logistics Coordinator',
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Oliver_Luo.png',
+      name: 'Oliver Luo',
+      emoji: '👴',
+      color: '#FFA500',
+      title: 'HackCamp Logistics Coordinator',
```

Comment:
Can you change Oliver's role to Sponsorship Coordinator

Reply:
done!